File: 0918/918.go
func maximumSumCircularSubarray(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := maximumSumCircularSubarray(root.Left)
	r := maximumSumCircularSubarray(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0918/918.c
int maximumSumCircularSubarray(struct TreeNode* root) {
    if (!root) return 0;
    int l = maximumSumCircularSubarray(root->left);
    int r = maximumSumCircularSubarray(root->right);
    return 1 + (l > r ? l : r);
}